src/elements/common/annotator-context/withAnnotatorContext.tsx

```diff
@@ -5,18 +5,20 @@ import { AnnotatorState, GetMatchPath } from './types';
 export interface WithAnnotatorContextProps {
     annotatorState?: AnnotatorState;
     emitAnnotatorActiveChangeEvent?: (id: string) => void;
+    emitRemoveEvent?: (id: string) => void;
     getAnnotationsMatchPath?: GetMatchPath;
 }
 
 export default function withAnnotatorContext<P extends {}>(WrappedComponent: React.ComponentType<P>) {
     return React.forwardRef<React.RefForwardingComponent<React.ComponentType<P>>, P>((props, ref) => (
         <AnnotatorContext.Consumer>
-            {({ emitActiveChangeEvent, getAnnotationsMatchPath, state }) => (
+            {({ emitActiveChangeEvent, emitRemoveEvent, getAnnotationsMatchPath, state }) => (
                 <WrappedComponent
                     ref={ref}
                     {...props}
                     annotatorState={state}
                     emitAnnotatorActiveChangeEvent={emitActiveChangeEvent}
+                    emitRemoveEvent={emitRemoveEvent}
```

Comment:
Should this be `emitAnnotatorRemoveEvent` to stay consistent with the the above?